helm-charts/dmr/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
I love these files (and don't change this now) - but could these be markdown?  Notes for future humans.

Reply:
Sure thing, just a little reformatting needed, there is an item here https://github.com/buerokratt/Infrastructure/issues/24 for further work on the helm charts I could capture this here :)